generators/docker/templates/src/main/docker/keycloak.yml.ejs

```diff
@@ -37,7 +37,7 @@ services:
       - 127.0.0.1:9080:9080
       - 127.0.0.1:9443:9443
     healthcheck:
-      test: ['CMD', 'curl', '-f', 'http://localhost:9080/realms/jhipster']
+      test: ['CMD',  'echo', '-n', 'GET /realms/jhipster HTTP/1.1\r\n\r\n', '|', 'nc', 'localhost', '9080']
```

Comment:
```suggestion
      test: ['CMD', 'nc', '-z', '-w2', 'localhost', '9080']
```

Reply:
In this case, we use the nc command with the -z option to scan for open ports, -w2 to specify a 2-second timeout, and localhost and 8080 as the host and port to check. If the connection fails or times out, nc will return a non-zero exit code, indicating an unhealthy state.

Reply:
Applied, please enable the checks to run again.